src/model/groups.rs

```diff
@@ -0,0 +1,116 @@
+use either::Either;
+use serde::{Deserialize, Serialize};
+
+use crate::id::User;
+
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+/// Details about a VRC group
+pub struct Group {
+	/// The unique identifier for the group.
+	pub id: String,
+	/// The name of the group.
+	pub name: String,
+	/// The short code associated with the group.
+	pub short_code: String,
+	/// The discriminator for the group.
+	pub discriminator: String,
+	/// The description of the group.
+	pub description: String,
+	/// The unique identifier for the group's icon.
+	pub icon_id: String,
+	/// The URL of the group's icon.
+	pub icon_url: String,
+	/// The unique identifier for the group's banner.
+	pub banner_id: String,
+	/// The URL of the group's banner.
+	pub banner_url: String,
+	/// The privacy setting of the group.
+	pub privacy: String,
+	/// The unique identifier of the owner of the group.
+	pub owner_id: String,
+	/// The rules associated with the group.
+	pub rules: String,
+	/// The list of links associated with the group.
+	pub links: Vec<String>,
+	/// The list of languages associated with the group.
+	pub languages: Vec<String>,
+	/// The count of members in the group.
+	pub member_count: i64,
+	/// The times tamp when the member count was last synchronized.
+	pub member_count_synced_at: String,
+	/// Indicates whether the group is verified.
+	pub is_verified: bool,
+	/// The join state of the group.
+	pub join_state: String,
+	// pub tags: Vec<Value>,
+	// pub galleries: Vec<Value>,
+	/// The time stamp when the group was created.
+	pub created_at: String,
+	/// The count of online members in the group.
+	pub online_member_count: i64,
```

Comment:
...I guess it indeed is better to support a case where the member counts are negative for whatever reason, even though on first thought it seemed wrong :smile:

Reply:
lol I think that's just what Json -> Serde defaulted to